### DISCUSSION TITLE
Cannot Load Model after Activating License Key

### DISCUSSION THREAD

**Discussion by Ahad-boop** [TabularEditor/TabularEditor3#791] [🙋♀️TE3 Q&A] [answered]
Hi! I have been using the TE3 for a month now as the trial. it worked perfectly so i switched to a license key now and activated it (after i was told my trial has ended) and now when i start it up and try to connect to the database. I cannot input the server address it doesnt let me do anything in that field. picture attached
![image](https://user-images.githubusercontent.com/61003451/211505851-11ac0d92-0cd4-4aa7-9662-04e24c7e704f.png)

**Comment by mlonsk** [comment, marked as answer]
Hi @Ahad-boop 
Is it correctly understood that you wish to connect to a Power BI XMLA endpoint or a Analysis Service instance (either on premise or in Azure)?
If so, it looks like you have purchase a Desktop license, that does not allow for this. This type of Tabular Editor 3 license can only connect to local instances of Power BI that you have open on your computer.
See https://docs.tabulareditor.com/te3/editions.html for further details on the different editions. 

All the best
Morten

**Reply by Ahad-boop** [reply]
Yes indeed Morten! I asked my colleague to buy the license and mentioned business but they got the desktop. I never double checked and got confused. Thank you so much for your answer and help!

**Reply by mlonsk** [reply]
Glad I could help.
Feel free to reach out through https://tabulareditor.com/contact if you need assistance upgrading your license to a Business edition. 

Just be aware that the Business license allows for connecting to Premium Per User workspaces in Power BI and to standard editions of SSAS and Azure Basic tier - I would just advice double checking that this is the type you need. I.e. if you have a Power BI Premium Dedicated Capacity you will need an Enterprise license. 

All the best
Morten
